src/styles.css

```diff
@@ -4,7 +4,7 @@
  * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap-blog-home/blob/master/LICENSE)
  */
 
- body {
+body {
```

Comment:
Good addition! looks better.

Reply:
what a change... lol

Reply:
LMAO

Reply:
LMAO I thought you had added way more, I'm clearly color blind....